### ISSUE TITLE
Release a v1.4.1 or v1.5.0 for go get

### ISSUE THREAD

**Issue by flibustenet** [phpdave11/gofpdf#2]
Currently
```sh
$ go get github.com/phpdave11/gofpdf
go get: github.com/phpdave11/gofpdf@v1.4.0: parsing go.mod:
	module declares its path as: github.com/jung-kurt/gofpdf
        but was required as: github.com/phpdave11/gofpdf
```
It works with `go get github.com/phpdave11/gofpdf@master`

But maybe you can release a v1.4.1 or v1.5.0 or change the current readme to add `@master`

**Comment by phpdave11** [comment]
I don't see where v1.4.0 is being referenced.  Also when I run the command:

```sh
go get github.com/phpdave11/gofpdf
```

I don't get any errors.  Can you provide more details?

**Comment by flibustenet** [comment]
The error comes with modules
```
mkdir test
cd test
go mod init test.tld
go get github.com/phpdave11/gofpdf
```
v1.4.0 is the most recent tag (then go get take this) and at this version go.mod is still with jung-kurt

**Comment by bkcsoft** [comment]
I'm having this problem as well which prohibits me for using it :|

```sh
# in a project using `go mod`
$ go get github.com/phpdave11/gofpdf/...
go: downloading github.com/phpdave11/gofpdf v1.4.0
go: found github.com/phpdave11/gofpdf/... in github.com/phpdave11/gofpdf v1.4.0
go get: github.com/phpdave11/gofpdf@v1.4.0: parsing go.mod:
        module declares its path as: github.com/jung-kurt/gofpdf
                but was required as: github.com/phpdave11/gofpdf
$ go version
go version go1.14.1 linux/amd64
```

**Comment by phpdave11** [comment]
Sorry about the delay.  I’ve released v1.4.1
